melodic/distribution.yaml

```diff
@@ -6325,6 +6325,23 @@ repositories:
       url: https://github.com/OUXT-Polaris/rostate_machine.git
       version: master
     status: developed
+  rosthrottle-release:
+    doc:
+      type: git
+      url: https://github.com/UTNuclearRoboticsPublic/rosthrottle-release.git
```

Comment:
```
ERROR: In file '''/home/travis/build/ros/rosdistro/melodic/distribution.yaml''': Could not validate doc entry for repo rosthrottle-release with error [[[Could not validate repository with url https://github.com/UTNuclearRoboticsPublic/rosthrottle-release.git and version 1.2.0 from entry at line 6329. Error reason: No branch found matching 1.2.0]]]
```

This has the release repo URL. Did you mean to put the source repo instead `https://github.com/UTNuclearRoboticsPublic/rosthrottle.git`?